tests/Api/Responses/admin/product_association_type/get_product_association_type_response.json

```
@@ -9,9 +9,8 @@
     "updatedAt": "@string@",
     "translations": {
       "en_US": {
-        "@id": "\/api\/v2\/admin\/product-association-type-translations\/@integer@",
+        "@id": "\/api\/v2\/admin\/product-association-types\/similar_products\/translations\/@string@",
```

```suggestion
        "@id": "\/api\/v2\/admin\/product-association-types\/similar_products\/translations\/en_US",
```
Here and other occurrences.